po/he.po

```diff
@@ -8745,7 +8745,7 @@ msgstr "חשבונאות לאנשים פרטיים ועסקים קטנים."
 #: gnucash/gnome-utils/gnc-main-window.c:4736
 msgid "translator-credits"
 msgstr ""
-"אבי מרקוביץ avi.markovitz@gmail.com 2021, 2020, 2019\n"
+"אבי מרקוביץ avi.markovitz@gmail.com 2022 2021, 2020, 2019\n"
```

Comment:
Welcome to the future! ;-)
How about;
אבי מרקוביץ avi.markovitz@gmail.com 2019–2021\n